modules/installation-configuration-parameters.adoc

```diff
@@ -3045,7 +3095,7 @@ Additional Nutanix configuration parameters are described in the following table
 |====
 [.small]
 --
-1. The `prismElements` section holds a list of Prism Elements (clusters). A Prism Element encompasses all of the Nutanix resources, for example virtual machines and subnets, that are used to host the {product-title} cluster. Only a single Prism Element is supported.
+1. The `prismElements` section holds a list of Prism Elements (clusters). A Prism Element encompasses all of the Nutanix resources, for example virtual machines and subnets, that are used to host the {product-title} cluster.
 2. Only one subnet per {product-title} cluster is supported.
```

Comment:
2. Only one subnet per Prism Element in an  {product-title} cluster is supported.